qa/L0_grpc_state_cleanup/cleanup_test.py

```diff
@@ -428,11 +431,18 @@ def test_simple_infer_timeout(self):
     def test_simple_infer_error_status(self):
         # This test case is used to check whether all the state objects are
         # released when RPC runs into error.
+        expected_exceptions = [
+            "This protocol is restricted, expecting header 'triton-grpc-protocol-infer-key'",
+            "The stream is no longer in valid state, the error detail is reported through provided callback. A new stream should be started after stopping the current stream.",
```

Comment:
Is `_simple_infer` using gRPC streaming?

Reply:
good catch

Reply:
removed